.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: php
 
+dist: trusty
+
 php:
-  - 5.3
-  - 5.4
-  - 5.5
```

Comment:
Can you re-add 5.5 please? Still supported according to composer.json.
We could remove it in a future version by updating the lib too.

Reply:
@mremi - I have an error when I added the PHP version `5.5`. I had to downgrade the `phpunit` package to version `4.8` (PHPUnit version 5 does not support PHP 5.5), and it gives this error: `Fatal error: Call to undefined method Mock_StopwatchEvent_7c17f876::stop() in /home/travis/build/mremi/UrlShortenerBundle/Provider/ProviderProxy.php on line 132`. I'm stuck here.